x-pack/plugins/canvas/public/components/shape_picker/shape_picker.tsx

```diff
@@ -9,9 +9,16 @@ import PropTypes from 'prop-types';
 import { EuiFlexGrid, EuiFlexItem, EuiLink } from '@elastic/eui';
 import { ShapePreview } from '../shape_preview';
 
-export const ShapePicker = ({ shapes, onChange }) => {
+interface Props {
+  shapes: {
+    [key: string]: string;
+  };
+  onChange?: (key: string) => void;
+}
+
+export const ShapePicker = ({ shapes, onChange = () => {} }: Props) => {
```

Comment:
(Apologies for the drive-by comment) In Cloud and EUI, we typically use the `FunctionComponent` type from `react`. This has the advantage that if a function returns something that React can't render, you'll get a type error. 

```javascript
export const ShapePicker: FunctionComponent<Props> = ({ shapes, onChange = () => {} }) => {
```

Reply:
This is my standard practice, as well... I think this was just an oversight on my part.  Thanks for letting me know!